targets/csharp/source/source/PlayFabHttp/PlayFabSysHttp.cs

```diff
@@ -12,6 +11,14 @@ namespace PlayFab.Internal
 {
     public class PlayFabSysHttp : IPlayFabHttp
     {
+		
+		private readonly HttpClient _client;
```

Comment:
Tabbing issues.

Reply:
I found the source of those nasty tabbing issues. Will fix em all

Reply:
fixed